### PR TITLE
[FEAT] objectstore (wip)

### PR DESCRIPTION
**Pull request by aricart**

### CODE REVIEW

**Comment by jmordica** [comment]
Woohoo! Excited about this one 🎉

File: tests/objectstore_test.ts [hunk 1]
```diff
@@ -0,0 +1,608 @@
+import { cleanup, jetstreamServerConf, setup } from "./jstest_util.ts";
```
Comment (kozlovic) [inline tests/objectstore_test.ts:1]: Do we need banner here?

**Review by kozlovic — APPROVED** [review]
LGTM

File: tests/objectstore_test.ts (hunk 1 above)
Comment (aricart) [reply tests/objectstore_test.ts:1]: Of course we do.

**Review by kozlovic — APPROVED** [review]
LGTM

**Review by kozlovic — APPROVED** [review]
LGTM

**Pull request merged**